library/stat.c

```diff
@@ -131,6 +131,13 @@ eblob_stat_global_print(FILE *fp, struct eblob_backend *b)
 		fprintf(fp, "%s: %" PRId64 "\n",
 				eblob_stat_get_name(b->stat, i),
 				eblob_stat_get(b->stat, i));
+	fprintf(fp, "datasort_total_data_size: %" PRId64 "\n",
```

Comment:
These statistics should also be added to [eblob_stat_global_json()](https://github.com/interiorem/eblob/blob/master/library/json_stat.cpp#L39) to make them available via monitor.

Reply:
done